src/modules/whats-new/components/whats-new-modal.tsx

```diff
@@ -58,6 +58,14 @@ export default function WhatsNewModal( { showModal, onClose }: WhatsNewModalProp
 				'Discover the latest updates in Studio! Explore exciting new features designed to enhance your experience.'
 			),
 		},
+		{
+			image: studioIllustration,
+			title: __( 'Introducing Studio by WordPress.com CLI' ),
+			description: __(
+				'Run studio command in your terminal to create, list, update, and delete preview sites with our new CLI tool.'
+			),
```

Comment:
```suggestion
			description: sprintf(
				/* translators: %s is the name of the Studio CLI command ("studio") */
				__(
					'Run the %s command in your terminal to create, list, update, and delete preview sites with our new CLI tool.'
				),
				'<code>studio</code>'
			),
```
Adding a `<code>` wrapper around "studio" is just a suggestion, but regarding the phrasing, I would definitely add a "the" before "studio command".

Reply:
We could also go back to your original approach of using backticks around the studio command name, @bcotrim. It looks better with a `<code>` wrapper, though.